src/libs/TransactionUtils.ts

```diff
@@ -315,6 +315,13 @@ function getOriginalAmount(transaction: Transaction): number {
     return Math.abs(amount);
 }
 
+/**
+ * Verify if the transaction is expecting the distance to be calculated on the server
+ */
+function hasPendingRoute(transaction: OnyxEntry<Transaction>): boolean {
```

Comment:
How about `hasPendingWaypoints`?

Reply:
Sure, that's better than `hasPendingRoute`. 

I don't think the word "pending" provides a whole lot of information about what's really happening. It would be hard for someone to know "Why would I use this method"? It would need a corresponding method description like you've got here already. If the name was `isFetchingWaypointsFromServer` then I don't think it would need a method description at all and is self-explanatory.